RapidReact/src/main/java/frc/robot/subsystem/IntakeSubsystem.java

```diff
@@ -107,6 +107,11 @@ public void toggle() {
     }
   }
 
+  public void forceIntaking() {
+    intakeSolenoid.set(Value.kForward);
```

Comment:
we have a boolean instance var that tracks the state of our intake. We need to set that here as well.